server/mqtt.go

```diff
@@ -2616,55 +2778,61 @@ func (as *mqttAccountSessionManager) transferUniqueSessStreamsToMuxed(log *Serve
 	retry = false
 }
 
-func (as *mqttAccountSessionManager) transferRetainedToPerKeySubjectStream(log *Server) bool {
+func (as *mqttAccountSessionManager) transferRetainedToPerKeySubjectStream(log *Server) error {
```

Comment:
We no longer check for error, so you could change that.

Reply:
0/5 prefer to keep since it makes sense for the function to return it. The caller is free to ignore, perhaps we do something else in the future with the error?